src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAYS = days;
+  const ONE_DAY_PRISE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCOUNT = 20;
+
+  if (DAYS >= LONG_TERM) {
+    const LONG_TERM_TOTAL = DAYS * ONE_DAY_PRISE - LONG_TERM_DISCOUNT;
+
+    return LONG_TERM_TOTAL;
+  }
+
+  if (DAYS >= MIDDLE_TERM) {
+    const MIDDLE_TERM_TOTAL = DAYS * ONE_DAY_PRISE - MIDDLE_TERM_DISCOUNT;
+
+    return MIDDLE_TERM_TOTAL;
+  }
```

Comment:
The logic for applying discounts is incorrect. If the number of days is greater than or equal to 7, both the 7-day and 3-day discounts should be considered. Currently, only the 7-day discount is applied. You should adjust the logic to ensure that the correct discount is applied based on the number of days. Consider using a single return statement at the end of the function to calculate the total cost after determining the applicable discount.